packages/playground/src/utils/grid.ts

```diff
@@ -12,7 +12,7 @@ export async function getGrid(profile: Pick<Profile, "mnemonic">, projectName?:
     backendStorageType: BackendStorageType.tfkvstore,
     projectName,
 
-    ...(import.meta.env.DEV
+    ...(network !== NetworkEnv.custom
```

Comment:
This won't pass custom urls in production mode
Condition should be update

```suggestion
    ...(import.meta.env.DEV && network !== NetworkEnv.custom
```